apps/meeting/README.MD

```diff
@@ -0,0 +1,43 @@
+# README - React Meeting Demo
+
+This is the SDK react meeting demo app. It shows how to use the Amazon Chime SDK and Chime SDK UI Component Library to build a react meeting app.
+
+## To run the demo app locally:
+
+1. If you haven't already, [configure your AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
+
+2. In the root directory `apps/meeting`, run `npm install`
+
+3. Start the webpack server and node server concurrently: `npm start`
+
+4. Open https://0.0.0.0:9000/ in your browser
```

Comment:
Is this 0.0.0.0:9000 or http://127.0.0.1:8080/ ?

Reply:
I tried 0.0.0.0:9000 and it works. Should I change it to http://127.0.0.1:8080/ ?

Reply:
I guess 0.0.0.0:9000 is fine. That is the one from webpack dev server.